hazelcast/src/test/java/com/hazelcast/jet/core/JetTestSupport.java

```diff
@@ -370,4 +372,22 @@ public static void cancelAndJoin(@Nonnull Job job) {
     public static <T> ProcessorMetaSupplier processorFromPipelineSource(BatchSource<T> source) {
         return ((BatchSourceTransform<T>) source).metaSupplier;
     }
+
+    /**
+     * Asserts that the job {@code job} has an {@link ExecutionContext} on the
```

Comment:
duplicate "job" phrases here. It seems that we can use these methods for normal jobs as well. I propose the changing the name of these to`assertJobExecutingOnMember`, wdyt about it?